docs/disks.md

```diff
@@ -0,0 +1,41 @@
+# RAID setup
+
+## Erasing existing metadata
+
+If your disks have been used in an array before, the metadata will need wiped before reusing them. **This will destroy the data on the disk**.
+
+```
+$ mdadm --misc --zero-superblock /dev/disk1
+```
+
+## Creating an array
+
+```
+mdadm --create \
+      --level=1 \
+      --metadata=1.0 \
+      --raid-devices=2 \
+      /dev/md/balena-raid \
+      /dev/disk1 \
+      /dev/disk2
+```
+
+Note that an MD device node will be automatically created/acquired by mdadm, and the path give under `/dev/md` will be symlinked to the device node.
+
+Substitute any other RAID level as required. Metadata 1.0 or older with mirroring is required to boot balenaOS. Newer formats place the metadata block at the beginning of the disk, which doesn't present a readable filesystem to the firmware before the array is assembled. This also applies to any RAID levels, such as striping and parity, that don't contain a mirror on each disk.
+
+## Installing balenaOS
+
+Once the array is assembled, balenaOS can be written directly to the MD device node:
+
+```
+$ dd if=balenaos.img of=/dev/md/balena-raid bs=4096
+```
```

Comment:
This needs to be amended as the flasher will now do this, right?

Reply:
@alexgg After reviewing your summary again, and testing creating an array with a name and the appropriate metadata version (0.90 doesn't support names), I'm going to modify the flasher script to write only to arrays with an appropriate name. I'll also update this doc with that information after it's working.

Reply:
ack